### ISSUE TITLE
Temporary User Blocking Plugin

### ISSUE THREAD

**Issue by DyteSprite** [powercord-community/suggestions#276]
### Description
Plugin should be able to allow for the temporary blocking of users. The plugin user should be able to configure a default time which they want to block the user for, as well as possibly override it. After the time expires, the user should be automatically unblocked.
### More info
For example, the default time configuration could be 12 hours (that the user sets), but they could override it to 24 hours or 6 hours for certain blocked users.
----

 - [x] I checked if this request wasn't already proposed
 - [x] I made sure this request follows Powercord's [plugin guidelines](https://powercord.dev/guidelines)

**Comment by dutake** [comment]
I gotchu, I think I can do this for ya

**Comment by Bowser65** [comment]
Duplicate of #36, prefer adding a comment on this issue instead
